docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+    mongodb-service:
+        build: ./mongodb
+        ports:
+            - 27017:27017
+    python-file:
+        build: ./pythonfile
+        ports:
+            - 8000:27017
```

Comment:
Please just do:

```
links:
    - mongodb-service
```
to automatically route all mongodb ports to only the python container and just open the port of the outgoing webserver port of flask.